### PR TITLE
created react app

### PR DESCRIPTION
**Pull request by eneadodi**
Should be working. Would like someone to confirm.

### CODE REVIEW

**Review by powellsl99 — APPROVED** [review]
LGTM. Nice job!

**Comment by david-fisher** [comment]
remember to associate pull requests with project (automation is marvelous), and assign reviewers and assignee. Reviewers approve, assigning merges after approval

**Pull request merged**
